src/frontend/js/components/PaymentButton/hooks/useTerms.tsx

```diff
@@ -66,12 +53,14 @@ export const useTerms = ({
             label={
               <>
                 {intl.formatMessage(messages.termsMessage)}{' '}
-                <button
-                  onClick={openContract}
+                <a
+                  href={context.site_urls.terms_and_conditions ?? '#'}
+                  target="_blank"
+                  rel="noopener noreferrer"
```

Comment:
In theory the contract will be hosted on our own infrastructure, why do you set those two `noopener noreffered`? Is this just a habit or for another specific reason?

Reply:
IMO with `target="_blank"` you set this `rel`. No matter the targeted url.